integration/builds/events/events_test.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"encoding/json"
+	"strings"
+	"testing"
+
+	"github.com/concourse/concourse/integration/internal/dctest"
+	"github.com/concourse/concourse/integration/internal/flytest"
+	"github.com/stretchr/testify/require"
+)
+
+func TestLoggingErrorEvents(t *testing.T) {
```

Comment:
Is there a reason you went with `integration` instead of `testflight` for this? To me, integration is more focused on testing behaviour under different deployment configurations, whereas testflight is more focused on pipeline/execution semantics - which feels like a more natural fit. testflight is also a fair bit lighter weight than integration

Reply:
For some reason, I had the thought that we weren't trying to write more `testflight` tests since adding `integration`. Looking over testflight I do see your point though and it seems like a fine place to put this test.

Reply:
Perhaps, though I thought the intention was for integration to effectively replace most of `topgun`. I think `testflight` still makes sense as an integration suite for things that are independent of Concourse configuration (e.g. things that should apply to all runtimes)

Reply:
Ah I think that's where the mix-up was in my head. topgun != testflight 😅